randomwallpaper@iflow.space/wallpaperController.js

```diff
@@ -403,21 +404,24 @@ var WallpaperController = class {
 		});
 	}
 
-	_getGeneralPostCommand(historyElement) {
-		let commandString = this._settings.get('general-post-command', 'string');
-
-		if (commandString === "") {
+	_getCommandArray(commandString, historyElement) {
+		let string = commandString;
+		if (string === "") {
 			return null;
 		}
 
-		let commandArray = commandString.split(' ');
+		// Replace variables
+		let variables = new Map();
+		variables.set('%wallpaper_path%', historyElement.path);
```

Comment:
Ah well, that already looks like a preparation for adding more variables. :smiley:

Reply:
I haven't actually added others but adding new ones should be pretty easy.